hack/update-gofmt.sh

```diff
@@ -27,20 +27,16 @@ kube::golang::verify_go_version
 
 cd "${KUBE_ROOT}"
 
-find_files() {
-  find . -not \( \
-      \( \
-        -wholename './output' \
-        -o -wholename './.git' \
-        -o -wholename './_output' \
-        -o -wholename './_gopath' \
-        -o -wholename './release' \
-        -o -wholename './target' \
-        -o -wholename '*/third_party/*' \
-        -o -wholename '*/vendor/*' \
-        -o -wholename './staging/src/k8s.io/client-go/*vendor/*' \
-      \) -prune \
-    \) -name '*.go'
+function git_find() {
+    # Similar to find but faster and easier to understand.  We want to include
+    # modified and untracked files because this might be running against code
+    # which is not tracked by git yet.
+    git ls-files -cmo --exclude-standard \
+        ':!:vendor/*' \
+        ':!:third_party/*' \
+        ':!:*/testdata/*' \
```

Comment:
should we exclude testdata

```
 git ls-files -cmo  --exclude-standard | grep testdata | grep \.go$
staging/src/k8s.io/kubectl/pkg/cmd/edit/testdata/record.go
test/conformance/testdata/embed.go
test/instrumentation/testdata/pkg/kubelet/metrics/metrics.go
test/instrumentation/testdata/staging/src/k8s.io/metrics/metrics.go
test/typecheck/testdata/bad/bad.go
test/typecheck/testdata/good/_ignore/bad.go
test/typecheck/testdata/good/good.go
test/typecheck/testdata/good/testdata/bad.go

```

Reply:
Good question - I wanted to retain the same behavior, but testdata is always questionable.  Will revert that part